tools/ci_connector_ops/ci_connector_ops/qa_checks.py

```diff
@@ -49,8 +51,7 @@ def check_migration_guide(connector: Connector) -> bool:
 
         if ordered_breaking_changes != ordered_heading_versions:
             print(f"Migration guide file for {connector.name} has missing or misordered version headings.")
-            print(f"Expected: {ordered_breaking_changes}")
-            print(f"Actual: {ordered_heading_versions}")
+            print(f"Expected headings: {ordered_expected_headings}")
```

Comment:
This felt easier to me - to distinguish exactly what headings i wanted instead of outputting just the versions.